bootloaders/encrypted/README.md

```diff
@@ -1,4 +1,6 @@
-For security you **must** replace private.pem and privateaes.bin with your own keys, and ivsalt.bin with your own per-device salt. Your signing key must be for the _secp256k1_ curve, in PEM format. You can create a .PEM file with:
+For security you **must** replace private.pem and privateaes.bin with your own keys, and ivsalt.bin with your own per-device salt. The AES key should also be different for each device. Make sure you **don't lose your keys and salts**, else you may not be able to boot code on your device.
```

Comment:
"boot code on your device" -> "update the code on your device" ?
(since I guess losing the keys won't stop the device booting from the code that's already in Flash!)